compiler/rustc_feature/src/active.rs

```diff
@@ -684,6 +684,9 @@ declare_features! (
     /// Allows `cfg(target_abi = "...")`.
     (active, cfg_target_abi, "1.55.0", Some(80970), None),
 
+    /// Allows `cfg(target_abi = "...")`.
```

Comment:
Doc comment needs to be updated here
```suggestion
    /// Allows for type equality constraints in `where` clauses.
```